docs/integrations/spark/installation.md

```diff
@@ -0,0 +1,255 @@
+---
+sidebar_position: 1
+title: Installation
+---
+
+:::warning
+
+* Version `1.8.0` and earlier only supported Scala 2.12 variants of Apache Spark.
+* Version `1.9.0` and later support both Scala 2.12 and 2.13 variants of Apache Spark.
+
+The above necessitates a change in the artifact identifier for `io.openlineage:openlineage-spark`.
+After version `1.8.0`, the artifact identifier has been updated. For subsequent versions, utilize:
+`io.openlineage:openlineage-spark_${SCALA_BINARY_VERSION}:${OPENLINEAGE_SPARK_VERSION}`.
+:::
+
+To integrate OpenLineage Spark with your application, you can:
+
+- [Bundle the package with your Apache Spark application project](#bundle-the-package-with-your-apache-spark-application-project).
+- [Place the JAR in your `${SPARK_HOME}/jars` directory](#place-the-jar-in-your-spark_homejars-directory)
+- [Use the `--jars` option with `spark-submit / spark-shell / pyspark`](#use-the---jars-option-with-spark-submit--spark-shell--pyspark)
+- [Use the `--packages` option with `spark-submit / spark-shell / pyspark`](#use-the---packages-option-with-spark-submit--spark-shell--pyspark)
+
+#### Bundle the package with your Apache Spark application project
+
+:::info
+This approach does not demonstrate how to configure the `OpenLineageSparkListener`.
+Please refer to the [Configuration](./configuration.md) section.
+:::
+
+For Maven, add the following to your `pom.xml`:
+
+<Tabs groupId="spark">
+<TabItem value="after-1.8.0" label="After 1.8.0">
```

Comment:
Not sure if the tabs work properly in this file, at least in the [preview](https://deploy-preview-290--mellifluous-eclair-326dcd.netlify.app/docs/integrations/spark/installation) here they did not render at all

Reply:
On localhost it looks the same, fyi. F.e. these are the tabs for scala and python: 
<img width="911" alt="image" src="https://github.com/OpenLineage/docs/assets/98873404/eb144434-9ec1-4959-9055-3e04a5b211c9">

Reply:
hmyy. perhaps I missed an import -> will look into this.